Psychtoolbox/PsychDemos/ProceduralSmoothedDiscsDemo.m

```diff
@@ -137,6 +146,8 @@ function ProceduralSmoothedDiscsDemo(ndiscs)
     % meaningful report of missed deadlines at the end of the script. Not
     % important for this demo, but here just in case you didn't know ;-)
     vbl = Screen('Flip', win, vbl + 0.5 * ifi);
+	
+	 count = count + 1;
```

Comment:
indentation

Reply:
Hopefully fixed (MATLAB's editor doesn't visualise whitespace, and I prefer to use tabs (=3 spaces), so I always end up manually changing settings so sometimes forget to do this)...